etc/grafana/dashboards/overview.json

```diff
@@ -39,7 +27,7 @@
       "type": "grafana",
       "id": "grafana",
       "name": "Grafana",
-      "version": "10.3.3"
+      "version": "10.1.1"
```

Comment:
is this coming from the public Grafana? if so, we need to bump it and re-export the dashboard again

Reply:
no from docker dashboard. I think the prev came from public.

Reply:
Let's get https://github.com/paradigmxyz/reth/pull/7032 in to minimize the amount of minor changes caused by different changes